docker/aarch64-release/compile.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+CMD="$1"
+
+if [ "${CMD}" == "--help" ]; then
+    echo "Usage: docker run <image> -ti release"
+    exit 0
+
+if [ "${CMD}" == "release" ]; then
+  echo "Release transport-native-epoll  and transport-native-unix-common for aarch64 platform"
+  # Update code
+  pushd netty
+  git pull
+  popd
+
+  # Cross compile for aarch64 platform
+  pushd netty/transport-native-unix-common
+  mvn clean install -Plinux-aarch64
+  popd
+  pushd netty/transport-native-epoll
+  mvn clean install -Plinux-aarch64 -DskipTests
+  popd
+
+  # TODO： Release the package to Maven center.
```

Comment:
I don't know how to release the package. Any suggestion?

Reply:
@wangxiyuan can you make all the docker / docker-compose stuff consistent with what we have atm ? If you don't have time I can also do it tho.

Reply:
I can but I don't know how to config the release management. So if you can take over this job. I'll be very appreciate. Thanks.

Reply:
I will

Reply:
Any progress, or anything elase I can do? I can update the dockerfile of cause.